src/lib/resolvers/index.ts

```diff
@@ -0,0 +1,23 @@
+export { resolveBoolean } from './boolean';
```

Comment:
Why not export all using `export * from './<resolver>';`?

Reply:
Also, I don't think this file is exported anywhere, so I don't know how will we use this 👀

Reply:
> Why not export all using export * from './<resolver>';?

I did not think of it. I'll commit.

> Also, I don't think this file is exported anywhere, so I don't know how will we use this 👀

Oops my bad :eyes: